tests/forte/data/data_store_serialization_test.py

```diff
@@ -0,0 +1,585 @@
+                    ][1],
+                    4567: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][2],
+                    5678: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][3],
+                    7890: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Document"
+                    ][4],
+                    9999: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][0],
+                    1234567: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][1],
+                    100: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][2],
+                    5000: temp._DataStore__elements[
+                        "ft.onto.base_ontology.Sentence"
+                    ][3],
+                },
+            )
+            print(f"deserialization without check attribute time cost {f-e}")
+
+            with self.assertRaisesRegex(
+                ValueError,
+                "Saved objects have unidentified fields, which raise an error.",
+            ):
+                DataStore.deserialize(
+                    tmpfilepath,
+                    serialize_method="json",
+                    check_attribute=True,
+                    accept_none=False,
+                )
+
+    def test_fast_pickle(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            tmpfilepath = os.path.join(tempdir, "temp2.txt")
+            a = timeit.timeit()
+            self.data_store.serialize(
+                tmpfilepath, serialize_method="json", save_attribute=False
+            )
+            b = timeit.timeit()
+            DataStore._type_attributes = {
+                "ft.onto.base_ontology.Document": {
+                    "attributes": {
+                        "document_class": 4,
+                        "sentiment": 5,
+                        "classifications": 6,
+                    },
+                    "parent_entry": "forte.data.ontology.top.Annotation",
+                },
+                "ft.onto.base_ontology.Sentence": {
+                    "attributes": {
+                        "speaker": 4,
+                        "part_id": 5,
+                        "sentiment": 6,
+                        "classification": 7,
+                        "classifications": 8,
+                    },
+                    "parent_entry": "forte.data.ontology.top.Annotation",
+                },
+            }
+            c = timeit.timeit()
+            temp = DataStore.deserialize(
+                tmpfilepath, serialize_method="json", check_attribute=False
+            )
+            d = timeit.timeit()
+            print(f"serialization without save attribute time cost {b-a}")
```

Comment:
we don't need to add the time profiling in test cases.

Reply:
fixed